modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,6 +15,136 @@
+
+[discrete]
+=== Online Demo
+
+Feel free to try Infinite Scale by exploring our online demo:
+
+- URL: https://ocis.owncloud.com/  
+- Username: `katherine`
+- Password: `gemini`
+
+Hint: You can also use your desktop, iOS or Android Apps to connect to our online demo for temporary evaluation purposes. 
+
+[discrete]
+#### How to Get the Rolling Release
+
+[discrete]
+##### Docker
+
+Pull the Docker image
+
+[source,shell]
+----
+docker pull owncloud/ocis-rolling:latest
+----
+
+[discrete]
+#### Binary
+
+Download the binary from our download server
+
+https://download.owncloud.com/ocis/ocis/rolling/[https://download.owncloud.com/ocis/ocis/rolling/]
+
+[discrete]
+#### Source Code on GitHub
+
+This release is tagged as `v6.0.0`` on GitHub
+
+https://github.com/owncloud/ocis/releases/tag/v6.0.0[https://github.com/owncloud/ocis/releases/tag/v6.0.0]
+
+[discrete]
+=== Web Office: Collabora Secure View
+
+Infinite Scale now supports Collabora’s Secure View feature, enhancing security for sensitive documents by allowing restricted viewing. Recipients can view content without downloading, copying, or editing, and a watermark with the user’s name is always applied to prevent unauthorized sharing. Secure View supports for example office documents, images, and PDFs. Read all about https://www.collaboraoffice.com/security/collabora-secure-view/[Collabora`s Secure View]. Collabora is the favored web office application of Infinite Scale, known for its strong focus on secure collaboration, making it ideal for organizations prioritizing data security and privacy.
+
+[discrete]
+=== Markdown Editor: ToastUI
+
+We have integrated the ToastUI markdown editor, providing users with robust capabilities to edit on markdown (.md or .markdown) files. Markdown files offer distinct advantages such as simplicity in formatting text using plain text syntax. The big advantage in markdown lies in its platform-independence as you don't need a complex or commercial App to edit these files. They are just plain simple and ultra useful. The ToastUI markdown editor enhances this experience with features like inline preview and syntax highlighting. Collaborative (live) editing is currently not available. 
+
+[discrete]
+=== View e-Books: ePub Reader
+Infinite Scale now includes per default the ability to open ePub e-books via the integrated ePub reader. This feature supports the EPUB 3.0 format as specified by the International Digital Publishing Forum (IDPF). It utilizes the Epub.js library, a versatile JavaScript tool for rendering ePub documents in the browser across various devices. Special thanks to https://github.com/dschmidt[dschmidt] for his valuable contribution.
```

Comment:
```suggestion
Infinite Scale now includes per default the ability to open ePub e-books via the integrated ePub reader. This feature supports the EPUB 3.0 format as specified by the International Digital Publishing Forum (IDPF). It utilizes the Epub.js library, a versatile JavaScript tool for rendering ePub documents in the browser across various devices.
```

If you want to mention someone, it was built by @AlexAndBear - same as the ToastUI integration. But honestly I wouldn't mention members of the core team. ;-)